components/description_list/description_list.stories.js

```diff
@@ -0,0 +1,123 @@
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+import DtDescriptionList from './description_list.vue';
+import DtDescriptionListDefaultTemplate from './description_list_default.story.vue';
+
+export const argTypesData = {
+  // Props
+  items: {
+    control: 'object',
+    table: {
+      defaultValue: {
+        summary: '{ term: string, description: string }[]',
+      },
+    },
+  },
+  descriptionClass: {
+    control: 'text',
+  },
+  termClass: {
+    control: 'text',
+  },
```

Comment:
Hmm because this is set to text the array you are passing into `WithStyles` is not showing up in the field. Maybe we set termClass control to object just for that case. Or pass it in as a string rather than an array.